includes/degree-functions.php

```diff
@@ -3,6 +3,378 @@
+ * @return string HTML markup
+ *
+ * // TODO Make this section dynamic
+ *
+ */
+
+ function get_degree_info_modern_layout( $post ) {
+
+	$program_type        = get_degree_program_type( $post );
+	$raw_postmeta        = get_post_meta( $post->ID );
+	$post_meta           = format_raw_postmeta( $raw_postmeta );
+	$colleges_list       = get_colleges_markup( $post->ID );
+	$departments_list    = get_departments_markup( $post->ID );
+	$industry_highlights = get_field('industry_highlights');
+	$promo_image         = get_field('promo_image');
+
+	ob_start();
+	?>
+	<div class="bg-faded">
+		<div class="container program-at-a-glance py-lg-3">
+			<div class="row my-lg-3">
+				<div class="col-lg-3 p-4 pb-3 mt-3 mt-lg-0">
+
+					<h2 class="h5 font-condensed text-uppercase">Program at a Glance</h2>
+					<dl class="mt-4 mt-lg-5">
+						<?php if ( $program_type ) : ?>
+						<dt class="h6 text-uppercase text-default">Program</dt>
+						<dd class="h6 mb-4"><?php echo $program_type->name; ?></dd>
+						<?php endif; ?>
+						<?php if ( $colleges_list ) : ?>
+						<dt class="h6 text-uppercase text-default">College(s)</dt>
+						<dd class="h6 mb-4"><?php echo $colleges_list; ?></dd>
+						<?php endif; ?>
+						<?php if ( $departments_list ) : ?>
+						<dt class="h6 text-uppercase text-default">Department(s)</dt>
+						<dd class="h6"><?php echo $departments_list; ?></dd>
+						<?php endif; ?>
+					</dl>
+				</div>
+
+				<div class="col-lg-2 text-center align-self-center">
+
+					<img class="icon-calendar img-fluid" style="max-height: 4em;" role="img"
+						src="https://www.ucf.edu/online/files/2019/12/Calendar-Icon-light-gray.svg" alt="">
+					<div class="h1 mt-2 mb-0">16</div>
+					<div class="h6 text-muted text-uppercase">Weeks</div>
+
+					<img class="icon-clock img-fluid" style="max-height: 4em;" role="img"
+						src="https://www.ucf.edu/online/files/2019/12/Clock-Icon-light-gray.svg" alt="">
+					<div class="h1 mt-2 mb-0">44</div>
+					<div class="h6 text-muted text-uppercase">Credit Hours</div>
+
+				</div>
+
+				<div class="col-lg-4 bg-secondary text-uppercase border-bottom mb-md-3">
+					<?php echo get_degree_tuition_markup( $post_meta, 'modern' ); ?>
+				</div>
+
+				<div class="col-lg-3 text-center align-self-center">
+					<?php if ( $promo_image ) : ?>
+					<img src="<?php echo $promo_image; ?>" class="img-fluid p-4" role="img">
+					<?php endif; ?>
+				</div>
+
+			</div>
+		</div>
+	</div>
+
+	<?php
+	return ob_get_clean();
+
+ }
+
+
+
+ /**
+ * Returns HTML for the degree description section.
+  *
+  * @since 3.4.0
+  * @author RJ Bruneel
+  * @return string HTML markup
+  *
+  */
+  function get_degree_description_modern_layout() {
+	$degree_description_modern_layout = get_field( 'degree_description_modern_layout' );
+
+	if( empty( $degree_description_modern_layout ) ) return '';
+
+	ob_start();
+	?>
+	<div class="container py-lg-3">
+		<div class="row my-lg-3">
+			<?php echo $degree_description_modern_layout; ?>
```

Comment:
Degree description needs to be contained in a `.col`., or the surrounding `.row` needs to be removed.